algorithms/install.sh

```diff
@@ -237,7 +239,7 @@ while [ !$finished ]; do
 
 	#if [[ $egs = *","* ]]; then
 		egs=${egs//,/|}
-		egs="(${egs})"
```

Comment:
() are needed, try to add it back and see what is the error

Reply:
try running the installation script the attached schema. 
Choose query #24

the result look like this for the edge _like_
t.@set_size_B__ = t.outdegree("(like)")

ERROR:
line 16:34 no viable alternative at input 'Start:s-((like'
Parsing encountered 1 syntax error(s)

Working version:
t.@set_size_B = t.outdegree("like")